### PR TITLE
Establish basic function.

### PR DESCRIPTION
**Pull request by mattcollier**
Addresses #1.  However, we are foregoing a pluggable architecture at this time.  We will rev up the lib as support for new DID methods are added.

A 1.0 release probably won't be made here until work is complete on new did-io and deps.  See: https://github.com/digitalbazaar/did-io/pull/50

### CODE REVIEW

**Review by dlongley — APPROVED** [review]
AFAICT, this is ok. @dmitrizagidulin does this work properly and do what we want with did-io in your view?

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Comment by dmitrizagidulin** [comment]
Looks good at first glance (though I haven't tested it yet).

File: lib/config.js [hunk 1]
```diff
@@ -7,4 +7,10 @@ const {config} = bedrock;
 const namespace = 'did-io';
 const cfg = config[namespace] = {};
 
-cfg.foo = '';
+cfg.methods = {
+  key: {},
+  v1: {
+    // modes: live, test
```
Comment (gannan08) [inline lib/config.js:13]:
```suggestion
    // modes: live, test, dev
```

**Comment by mattcollier** [comment]
Waiting on https://github.com/digitalbazaar/did-method-key-js/pull/35.

**Comment by dmitrizagidulin** [comment]
@mattcollier `@digitalbazaar/did-method-key` now published as `v1.0.0`! :)

**Comment by mattcollier** [comment]
@dmitrizagidulin congratulations!  Updating.

**Pull request merged**